homeassistant/components/dio_chacon/entity.py

```diff
@@ -0,0 +1,56 @@
+"""Base entity for the Dio Chacon entity."""
+
+import logging
+from typing import Any
+
+from dio_chacon_wifi_api import DIOChaconAPIClient
+
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import DOMAIN, MANUFACTURER
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class DioChaconEntity(Entity):
+    """Implements a common class elements representing the Dio Chacon entity."""
+
+    _attr_should_poll = False
+    _attr_has_entity_name = True
+
+    def __init__(
+        self, dio_chacon_client: DIOChaconAPIClient, device: dict[str, Any]
+    ) -> None:
+        """Initialize Dio Chacon entity."""
+
+        self.dio_chacon_client: DIOChaconAPIClient = dio_chacon_client
```

Comment:
```suggestion
        self.client = dio_chacon_client
```
Maybe makes it a bit more readable

Reply:
Done